packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -1192,6 +1248,8 @@ export function createProjectionNode<I>({
 
                 this.notifyListeners("projectionUpdate", target)
             }
+
+            projectionFrameData.recalculatedProjection++
```

Comment:
What happens here? Can we add a comment?

Reply:
Added comments to all debug counters